pkg/kubectl/scale.go

```diff
@@ -424,8 +431,11 @@ func (scaler *DeploymentScaler) ScaleSimple(namespace, name string, precondition
 
 	// TODO(madhusudancs): Fix this when Scale group issues are resolved (see issue #18528).
 	// For now I'm falling back to regular Deployment update operation.
```

Comment:
Yeah, this should now work as long as the resources are RCs, RSs, Deployments or HPA.